salt/bugs/config/nginx.conf.jinja

```diff
@@ -77,3 +77,49 @@ server {
     proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
   }
 }
+
+server {
+  listen {{ port }} ssl;
+  server_name {{ server_name }};
+  include mime.types;
+
+  ssl_certificate /etc/ssl/private/bugs.psf.io.pem;
+  ssl_certificate_key /etc/ssl/private/bugs.psf.io.pem;
```

Comment:
```suggestion
  ssl_certificate /etc/ssl/private/bugs.python.org.pem;
  ssl_certificate_key /etc/ssl/private/bugs.python.org.pem;
```

Reply:
@ewdurbin ?

Reply:
I don't think so. Since we are standing up _new_ server blocks for internal access on ports 9000-9002 and having them validate using the internal TLS, we want to use the internal TLS cert for bugs.psf.io.

Reply:
😵‍💫